### ISSUE TITLE
Change connect() to return Promise<void>

### ISSUE THREAD

**Issue by avayvod** [w3c/remote-playback#45]
... and reject when user cancels the selection.
Having to wait for the Promise to be resolved and then to check if its value is true seems to make the examples less straightforward:

``` javascript
videoElem.remote.connect().then(switchToRemoteUI);
```

vs.

``` javascript
videoElem.remote.connect().then(function(success) {  if (success) switchToRemoteUI(); });
```

**Comment by avayvod** [comment]
Or return `Promise<RemotePlaybackState>` since the method could be used to both start and stop remote playback as per issue #4

**Comment by avayvod** [comment]
@mounirlamouri since you were for `Promise<bool>` in the first place :P

**Comment by avayvod** [comment]
Offline we agreed on returning a `Promise<void>`

**Comment by avayvod** [comment]
I believe this is fixed with #49
